src/client/client.cpp

```diff
@@ -1344,25 +1344,19 @@ void Client::setPlayerControl(PlayerControl &control)
 void Client::selectPlayerItem(u16 item)
 {
 	m_playeritem = item;
-	m_inventory_updated = true;
+	m_update_wielded_item = true;
 	sendPlayerItem(item);
 }
 
 // Returns true if the inventory of the local player has been
 // updated from the server. If it is true, it is set to false.
-bool Client::getLocalInventoryUpdated()
+bool Client::updateWieldedItem()
```

Comment:
There seems to be a bunch of outdated comments?

Reply:
No, it's still valid.